tools/test/examples/update.py

```diff
@@ -431,6 +433,47 @@ def create_work_directory(path):
     
     os.makedirs(path)
 
+def check_update_status(examples, github, tag):
+    """ Check the status of previously raised update pull requests
+    
+    Args:
+    examples - list of examples which should have had PRs raised against them. 
+    github - github rest API instance
+    tag - release tag used for the update
+    
+    """
+
+    for example in examples:
+
+        repo_name = ''.join(['ARMmbed/', example['name']])
+        try:
+            repo = github.get_repo(repo_name, False)
+
+        except Exception as exc:
+            text = "Cannot access: " + str(repo_name)
+            userlog.error(text)
+            userlog.exception(exc)
+            sys.exit(1)
+
+        # Create the full repository filter component
+        org_str = ''.join(['repo:', 'ARMmbed/', example['name']])
```

Comment:
Why are `'repo:' and 'ARMmbed/' separate strings here? They are just going to be concatenated anyway.

Reply:
yeah , very minor but I could fix it...